start-configuration

```diff
@@ -45,6 +45,26 @@ if [[ $PROXY ]]; then
     sleep 5
 fi
 
+if [[ $RCON_PASSWORD_FILE ]]; then
+    log ""
+    if [ ! -e ${RCON_PASSWORD_FILE} ]; then
+	log "Initial RCON password file ${RCON_PASSWORD_FILE} does not seems to exist."
+	log "Please ensure your configuration."
+	log "If you are using Docker Secrets feature, please check this for further information: "
+	log " https://docs.docker.com/engine/swarm/secrets"
+	log "ENV value RCON_PASSWORD would be used."
+	log ""
```

Comment:
I would prefer it exit immediately with an error since falling back to `RCON_PASSWORD` would be very misleading if someone intended to use `RCON_PASSWORD_FILE`

```suggestion
  log ""
	exit 1
```